src/api/connect.js

```diff
@@ -43,8 +45,13 @@ export default options => {
     }
 
     // create a new socket
-    if (socket === null) {
-      socket = new WebSocket(makeWebsocketAddress(options), protocols)
+    if (socket == null) {
+      if (options.debug) { 
+        console.log('Opening socket to ' + socketAddress)
+      }
+      let protocols = (options && options.protocol) || 'notification'
```

Comment:
Renamed the new option "subprotocols" instead of "protocols". It caused confusion with other option called "protocol". In fact, I had a typo in this line because of that.
Also added the "subprotocols" option in readme.md.